src/main/java/com/kyonggi/teampu/domain/admin/service/AdminService.java

```diff
@@ -29,6 +34,13 @@ public List<AppliedInfoResponse> getHome(){
             .toList();
     }
 
+    public void approve(ApproveRequest request, Member connectedMember){
+        if (!connectedMember.getIsAdmin()) throw new RuntimeException(NOT_ADMIN.getMessage());
+
+        Application application = applicationRepository.findById(request.getApplicationId()).get();
+        application.updateStatus(request.getStatus());
```

Comment:
```suggestion
        application.updateStatus(ApplicationStatus.APPROVED);
```

어차피 승인만 처리할 거면 status를 request dto에서 받아오지 않고 그냥 요렇게 고정해놔도 될 것 같습니당

Reply:
해당 API 하나로 승민 + 거부를 전부 다 처리하도록 구현해서, request에서 REJECTED를 넣어주면 거부 처리가 됩니당

Reply:
아하... 그러면 메서드명을 updateStatus() 이런 걸로 하면 승인/거부 다 가능한 느낌이니까 더 좋을 듯요!

Reply:
엄.. 그렇네요! 수정하겠슴다!